lib/verdict/experiment.rb

```diff
@@ -211,14 +198,6 @@ def to_json(options = {})
     as_json(options).to_json
   end
 
-  def fetch_subject(subject_identifier)
-    raise NotImplementedError, "Fetching subjects based in identifier is not implemented for eperiment @{handle.inspect}."
-  end
-
-  def fetch_assignment(subject_identifier)
```

Comment:
i want to keep `fetch_assignment(subject)` being the one that calls `@storage.retreive_assignment`, and i want `lookup(subject)` to basically just call `fetch_assignment(subject)`.
Please also change back the methods that you changed from `fetch_assignment` to `lookup`